packages/block-library/src/link-preview/transforms.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Default transforms for generic embeds.
+ */
+const transforms = {
```

Comment:
I tried pasting `https://aheadcreative.com` into the editor canvas and it turned it into an embed block. I couldn't see how to get the link preview block to work.

Reply:
Oh, I've not added that yet. When should we convert a pasted link to a link preview? When it fails to embed?

Reply:
If that can be achieved then yes that would seem logical 👍